Library/ViewModels/DiscoveryPageViewModel.swift

```diff
@@ -200,7 +200,6 @@ public final class DiscoveryPageViewModel: DiscoveryPageViewModelType, Discovery
       isRefreshing.map { ($0, true) },
       projectsLoadingNoRefresh.map { ($0, false) },
       self.viewWillAppearProperty.signal.take(first: 1).mapConst((true, false))
-        .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
```

Comment:
Decided to go with `DispatchQueue.main.async { ... }` in the VC instead. It was the only way to get the snapshots to reliably record correctly 🤷‍♂.